internal/controller/cloud-control/iprange_azure_test.go

```diff
@@ -205,6 +207,36 @@ var _ = Describe("Feature: KCP IpRange for Azure", func() {
 			Expect(azureSecurityGroup).NotTo(BeNil())
 		})
 
+		var virtualNetworkLink *armprivatedns.VirtualNetworkLink
```

Comment:
You don't need eventually here. Just move it after https://github.com/kyma-project/cloud-manager/pull/687/files#diff-c52b7ce9733d241804c6e5e720b8fcb2b519c25f189ed988c2a8cb5be97a5f06R263